res/xml/urdu phonetic

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
The layout file must have a specific name: `<script>_<layout name>_<variant>.xml`.
`urdu_phonetic_ur.xml` is an acceptable name.

Once you have corrected the name, update generated files by running `make gen_layouts`.